pfsimulator/parflow_lib/vector_utilities.c

```diff
@@ -2083,6 +2083,6 @@ void PFVLayerCopy(
   IncFLOPCount(2 * VectorSize(x));
 }
 
-#ifdef __cpluspus
+#ifdef __cplusplus
```

Comment:
This may have some code functionality changes.

Reply:
Yes, this causes an error:
```
[ 62%] Building CXX object pfsimulator/parflow_lib/CMakeFiles/pfsimulator.dir/vector_utilities.cpp.o
/home/runner/work/parflow/parflow/pfsimulator/parflow_lib/vector_utilities.cpp:23:1: error: expected declaration before ‘}’ token
   23 | }
      | ^
```
should this `#ifdef/#endif` block be removed altogether?